bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/utils/messages.properties

```diff
@@ -122,6 +122,7 @@ preferences_saveProblems=Exception occurred while saving project preferences: {0
 preferences_syncException=Exception occurred while synchronizing node: {0}.
 
 updateUnknownNatureMarkers=Update markers for unknown nature on projects
+updateMissingProjectEncodingMarkers="Update markers for missing project encoding
```

Comment:
This is not used anymore, please remove

Reply:
removed line (=> no further changes => this class is no longer listed in the PR)